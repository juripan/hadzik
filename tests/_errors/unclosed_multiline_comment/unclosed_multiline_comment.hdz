/*
oops forgor to close the comment ohhh nooo

