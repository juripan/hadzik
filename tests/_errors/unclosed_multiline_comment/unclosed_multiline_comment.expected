Failed here: tests/_errors/unclosed_multiline_comment/unclosed_multiline_comment.hdz:1:1
/*
^
[31mSyntaxError[0m: (line 1 column 1) unclosed multiline comment
